CommonLibF4/include/RE/Bethesda/MemoryManager.h

```diff
@@ -289,7 +289,10 @@ namespace RE
 
 	[[nodiscard]] inline void* calloc(std::size_t a_num, std::size_t a_size)
 	{
-		return malloc(a_num * a_size);
+		auto ret = malloc(a_num * a_size);
```

Comment:
```suggestion
		const auto ret = malloc(a_num * a_size);
```
Please always declare local variables `const` when appropriate

Reply:
Done